smart-contracts/scripts/deployments/keyManager.js

```diff
@@ -0,0 +1,49 @@
+const { ethers, upgrades } = require('hardhat')
+const { networks } = require('@unlock-protocol/networks')
+const { getNetworkName } = require('../../helpers/network')
+
+async function main(locksmiths) {
+
+  const [deployer] = await ethers.getSigners()
+
+  // fetch chain info
+  const chainId = await deployer.getChainId()
+  const networkName = getNetworkName(chainId)
+
+  const KeyManager = await ethers.getContractFactory('KeyManager')
+  const keyManager = await upgrades.deployProxy(KeyManager, [], {
+    initializer: 'initialize()',
+  })
+  await keyManager.deployed()
+  console.log(`> Deployed KeyManager on ${networkName} to ${keyManager.address}`)
+
+  // Add locksmiths
+  for (const locksmith of locksmiths) {
+    await keyManager.addLocksmith(locksmith)
+    console.log(`> Added ${locksmith} as signer on KeyManager`)
+  }
+
+  // Transfer ownership to multisig (based on networks package)
+  if (networks[chainId.toString()].multisig) {
+    await keyManager.transferOwnership(networks[chainId.toString()].multisig)
+    console.log(`> Transfered ownership of KeyManager to ${networks[chainId.toString()].multisig}`)
+  }
+
+  // TODO: transfer proxy admin to multisig (based on networks package?)
```

Comment:
@clemsos Any idea how to do that??

Reply:
yes we have a script for that already https://github.com/unlock-protocol/unlock/blob/master/smart-contracts/scripts/multisig/transferOwnership.js

Also you can use the CLI directly `yarn hardhat gnosis:transfer` if that helps.

Reply:
proxyAdmin is here, you need ot transfer ownership  https://goerli.etherscan.io/address/0x604fa8712e61fe65db326f97905efcb087c631cc#writeContract

Reply:
I added a few things to the script.